### PR TITLE
shortened passphrases to reduce typo errors

### PR DESCRIPTION
**Pull request by mbiarnes**
People start to complain that the pass phrase are very long and that the "please" is superfluous.
I think of shorten the pass phrases.
**IMPORTANT**: before merging thiswe have to send a mail to BSIG with this new pass phrases.

### CODE REVIEW

**Comment by Ginxo** [comment]
I more or less agree with the `please`, but I don't see the point to short the rest

**Comment by mareknovotny** [comment]
I don'ŧ insist on please, it can be just optional so any polite person can add that, but it should not be requested there

File: job-dsls/jobs/new_compile_downstream_build.groovy [hunk 1]
```diff
@@ -127,7 +127,7 @@ for (repoConfig in REPO_CONFIGS) {
                 orgslist("${ghOrgUnit}")
                 whitelist("")
                 cron("")
-                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) compile downstream build.*")
+                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) compile db.*")
```
Comment (mareknovotny) [inline job-dsls/jobs/new_compile_downstream_build.groovy:130]: if we would like to get shorter one then I suggest:
```suggestion
                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) cdb.*")
```
or
```suggestion
                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) full compile.*")
```

File: job-dsls/jobs/new_downstream_pr_jobs.groovy [hunk 2]
```diff
@@ -124,7 +124,7 @@ for (repoConfig in REPO_CONFIGS) {
                 orgslist("${ghOrgUnit}")
                 whitelist("")
                 cron("")
-                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) full downstream build.*")
+                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) full db.*")
```
Comment (mareknovotny) [inline job-dsls/jobs/new_downstream_pr_jobs.groovy:127]: if we would like to get shorter one then I suggest:
```suggestion
                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) fdb.*")
```

File: job-dsls/jobs/new_downstream_production.groovy [hunk 3]
```diff
@@ -120,7 +120,7 @@ for (repoConfig in REPO_CONFIGS) {
                 orgslist("${ghOrgUnit}")
                 whitelist("")
                 cron("")
-                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) full downstream production build.*")
+                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) prod downstream.*")
```
Comment (mareknovotny) [inline job-dsls/jobs/new_downstream_production.groovy:123]: if we would like to get shorter one then I suggest:
```suggestion
                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) full prod build.*")
```
or
```suggestion
                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) product FDB.*")
```

File: job-dsls/jobs/new_pr_jobs.groovy [hunk 4]
```diff
@@ -201,7 +201,7 @@ for (repoConfig in REPO_CONFIGS) {
                 orgslist("${ghOrgUnit}")
                 whitelist("")
                 cron("")
-                triggerPhrase(".*[j|J]enkins,?.*(retest|test) this please.*")
+                triggerPhrase(".*[j|J]enkins,?.*(retest|test) this.*")
```
Comment (mareknovotny) [inline job-dsls/jobs/new_pr_jobs.groovy:204]: this is ok to remove please word even to be polite you can still include it no matter what phrase reges is ;)

File: job-dsls/jobs/new_upstream.groovy [hunk 5]
```diff
@@ -120,7 +120,7 @@ for (repoConfig in REPO_CONFIGS) {
                 orgslist("${ghOrgUnit}")
                 whitelist("")
                 cron("")
-                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) upstream build.*")
+                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) upstream.*")
```
Comment (mareknovotny) [inline job-dsls/jobs/new_upstream.groovy:123]: thinking why we have this to be triggered by comment phrase?
I don't see a reason for it or please explain it to me ;)

File: job-dsls/jobs/new_downstream_production.groovy (hunk 3 above)
Comment (AntStephenson) [reply job-dsls/jobs/new_downstream_production.groovy:123]: I guess option B but lowercase to be consistent with the others.

**Comment by AntStephenson** [comment]
I quite like the short hand suggests by @mareknovotny

**Comment by Ginxo** [comment]
Well from my point of view there's no reason to change "launch" expressions, I think it will generate problems for the ones already use to them. In case we want them in any case I would suggest changes like

- `.*[j|J]enkins,?.*(execute|run|trigger|start|do) (prod|product) (downstream|FDB).*`
- `.*[j|J]enkins,?.*(execute|run|trigger|start|do) (full downstream build|full db@FDB).*`
and so on... So it's just to support old ones and shorter options.

I wouldn't go for `please` removing since the addition to default trigger phrase solved the problem of triggering everything

**Comment by mbiarnes** [comment]
close because the PR #803 does this

**Pull request closed without merging**